src/Controller/ReminderController.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\CaseEntity;
+use App\Entity\Reminder;
+use App\Entity\User;
+use App\Form\ReminderType;
+use App\Repository\ReminderRepository;
+use App\Service\ReminderHelper;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Security;
+
+class ReminderController extends AbstractController
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+    /**
+     * @var ReminderHelper
+     */
+    private $reminderHelper;
+    /**
+     * @var Security
+     */
+    private $security;
+
+    public function __construct(EntityManagerInterface $entityManager, ReminderHelper $reminderHelper, Security $security)
+    {
+        $this->entityManager = $entityManager;
+        $this->reminderHelper = $reminderHelper;
+        $this->security = $security;
+    }
+
+    /**
+     * @Route("/reminder", name="reminder_index")
+     */
+    public function index(ReminderRepository $reminderRepository): Response
+    {
+        /** @var User $user */
+        $user = $this->security->getUser();
+
+        $reminders = $reminderRepository->findBy([
+            'createdBy' => $user->getId()->toBinary(),
+        ]);
+
+        return $this->render('reminder/index.html.twig', [
+            'reminders' => $reminders,
+        ]);
+    }
+
+    /**
+     * @Route("/reminder/create/{id}", name="reminder_create", methods={"GET", "POST"})
+     */
+    public function newReminder(CaseEntity $case, Request $request): Response
```

Comment:
Consider renaming the method to just `new` as its implicit its related to a reminder.

Reply:
Renamed to `new`.